injector_test.py

```diff
@@ -1583,6 +1584,43 @@ def test_binder_has_implicit_binding_for_implicitly_bound_type():
     assert not injector.binder.has_explicit_binding_for(int)
 
 
+def test_injector_with_uniqueness_checking_raises_error():
+    def configure(binder):
+        binder.bind(int, to=123)
+        binder.bind(int, to=456)
+
+    with pytest.raises(NonUniqueBinding, match="Binding for 'int' already exists"):
+        _ = Injector([configure], unique=True)
```

Comment:
Do we want a test for the opposite case, where overriding works without `unique=True`?

Reply:
@cswartzvi bump?

Reply:
@craig8  apologies, I completely forgot about this PR. I will try and push the requested changes this weekend.